app/gui/view/welcome-screen/src/side_menu.rs

```diff
@@ -64,6 +69,37 @@ impl SideMenu {
         };
         root_dom.append_or_warn(&projects_list, &logger);
 
-        Self { logger, root_dom, projects_list, new_project_button, closures: default() }
+        Self { logger, root_dom, frp, projects_list, new_project_button, closures: default() }
+    }
+
+    pub fn update_projects_list(&self, projects: &[String]) {
```

Comment:
This one might be clearer as `set_project_list`.

Reply:
Fixed as well